test/core/Safe.Execution.spec.ts

```diff
@@ -53,21 +53,27 @@ describe("Safe", async () => {
             const { safe } = await setupTests();
             const tx = buildSafeTransaction({ to: safe.address, safeTxGas: 1000000, nonce: await safe.nonce() });
             const signatureBytes = buildSignatureBytes([await safeApproveHash(user1, safe, tx, true)]);
-            await expect(
-                safe.execTransaction(
-                    tx.to,
-                    tx.value,
-                    tx.data,
-                    tx.operation,
-                    tx.safeTxGas,
-                    tx.baseGas,
-                    tx.gasPrice,
-                    tx.gasToken,
-                    tx.refundReceiver,
-                    signatureBytes,
-                    { gasLimit: 1000000 },
-                ),
-            ).to.be.revertedWith("GS010");
+
+            const txPromise = safe.execTransaction(
+                tx.to,
+                tx.value,
+                tx.data,
+                tx.operation,
+                tx.safeTxGas,
+                tx.baseGas,
+                tx.gasPrice,
+                tx.gasToken,
+                tx.refundReceiver,
+                signatureBytes,
+                { gasLimit: 1000000 },
+            );
+
+            // Reverted reason seems not properly returned by zkSync local node, though it is in fact GS010 when using debug_traceTransaction
```

Comment:
can this be extracted into a utility function? e.g., `assertZksyncRevert` with message that would do this under the hood?

Reply:
The test only checks that the transaction is reverted and doesn't check the revert message. However, a comment says that you can use `deub_traceTransaction` to verify that it is in `GS010`. Is there a way to write a utility function that would call `debug_traceTransaction` under the hood and verify the error message?